README.md

```diff
@@ -1,33 +1,64 @@
-# Heroku-transparent-up [![Build Status](https://secure.travis-ci.org/noblesamurai/heroku-transparent-up.png?branch=master)](http://travis-ci.org/noblesamurai/heroku-transparent-up) [![NPM version](https://badge-me.herokuapp.com/api/npm/heroku-transparent-up.png)](http://badges.enytc.com/for/npm/heroku-transparent-up)
+# Heroku-formation [![Build Status](https://secure.travis-ci.org/noblesamurai/heroku-formation.png?branch=master)](http://travis-ci.org/noblesamurai/heroku-formation) [![NPM version](https://badge-me.herokuapp.com/api/npm/heroku-formation.png)](http://badges.enytc.com/for/npm/heroku-formation)
 
-> Proxies to a heroku web dyno, starting up required dynos transparently.
+> Enacts a requested dyno formation on heroku across multiple apps.
 
 ## Purpose
-- What problem does this module solve? At least a few sentences.
-PLEASE_FILL_IN_HERE
+Allows you to request a dyno formation across multiple micro-services and wait
+until it is effective.
 
-## Usage
+## Installation
+```bash
+$ npm install heroku-formation
+```
 
+## Usage
 ```js
-// Several examples of usage.
-// Usually copying and pasting code from the tests and making the code standalone suffices.
-// PLEASE_FILL_IN_HERE
+const Heroku = require('heroku-client');
+const heroku = new Heroku({ token: 'mytoken' });
+const { applyFormation, checkFormation } = require('heroku-formation');
+
+async function main () {
+  await applyFormation(heroku, 'appName', [{ type: 'web', quantity: 1 }]);
+  // We get here after formation is enacted.
+  const result = await checkFormation(heroku, 'appName', [{ type: 'web', quantity: 2 }]);
+  // result is false
+}
 ```
 
+The formation should be of the format:
+{
+    web: formation1,
+    worker: formation2,
+        /*...*/
+}
+where the formation is compatible with [this doco](https://devcenter.heroku.com/articles/platform-api-reference#formation).
```

Comment:
maybe show an example of an actual formation above as well as pointing at the docs